app/main.py

```diff
@@ -1,6 +1,13 @@
-# write your imports here
+from app.cinema.bar import CinemaBar
+from app.cinema.hall import CinemaHall
+from app.people.cinema_staff import Cleaner
+from app.people.customer import Customer
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(customers, hall_number, cleaner, movie):
+    ls_of_cls_obj = [Customer(cus["name"], cus["food"]) for cus in customers]
```

Comment:
```suggestion
    ls_of_cls_obj = [Customer(cus["name"], cus["food"]) for cus in customers]
```
or maybe Customer objects?